.pre-commit-config.yaml

```diff
@@ -1,5 +1,38 @@
 repos:
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: v0.0.255
+    hooks:
+      - id: ruff
+        args: ["--config", "pyproject.toml"]
```

Comment:
This makes sure it uses the top one and not the one in the template. See charliermarsh/ruff#1812.

Reply:
that's handy to know!